### PR TITLE
Added (commented) case for windows

### PR DESCRIPTION
**Pull request by rougier**
I've been using this example during a lesson and it was not working on windows.
I had to use the added (commented) code. Unfortunately, I cannot test it anymore (I've been tested it on a student machine).

### CODE REVIEW

**Comment by rougier** [comment]
ping !

**Comment by rougier** [comment]
ping !

File: advanced/interfacing_with_c/ctypes/cos_module.py [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+# libm = cdll.msvcrt
+
+
 # set the argument type
 libm.cos.argtypes = [ctypes.c_double]
 # set the return type
 libm.cos.restype = ctypes.c_double
 
 
+
```
Comment (GaelVaroquaux) [inline advanced/interfacing_with_c/ctypes/cos_module.py]: That's not PEP8 :(

**Comment by GaelVaroquaux** [comment]
Thanks, merging.

**Pull request merged**